code/game/objects/items/robot/robot_parts.dm

```diff
@@ -78,6 +78,52 @@
 			else
 				to_chat(user, "<span class='warning'>You need one sheet of metal to start building ED-209!</span>")
 				return
+	else if(istype(W, /obj/item/wrench)) //Deconstucts empty borg shell. Flashes remain unbroken because they haven't been used yet
+		var/turf/T = get_turf(src)
+		src.forceMove(T)
+		if(l_leg || r_leg || chest || l_arm || r_arm || head)
+			if(W.use_tool(src, user, 5, volume=50))
+				if(l_leg)
+					src.l_leg.forceMove(T)
+					src.l_leg = null
+				if(r_leg)
+					src.r_leg.forceMove(T)
+					src.r_leg = null

```

Comment:
You don't need all these `src.`

You can in fact search and replace that in your additions

Reply:
Yep, looks like it runs fine without `src`. Why is that even there? Oh well, updated